recipe/meta.yaml

```diff
@@ -25,10 +25,11 @@ requirements:
   build:
     - python
     - setuptools
+    - pytest-runner
 
   run:
     - python
-    - astroid >=1.4.5,<1.5.0
+    - astroid >=1.5.1,<1.6.0
```

Comment:
Doesn't appear to be [upper bounded]( https://github.com/PyCQA/pylint/blob/pylint-1.7.1/pylint/__pkginfo__.py#L25 ).